src/apis/CameraRoll.re

```diff
@@ -20,46 +20,46 @@ external getPhotosParams:
     ~mimeTypes: array(string)=?,
     unit
   ) =>
-  getPhotosParams =
-  "";
+  getPhotosParams;
 
-type photoIdentifier = {
-  .
-  "node": {
-    .
-    "_type": string,
-    "group_name": string,
-    "image": {
-      .
-      "filename": string,
-      "uri": string,
-      "height": float,
-      "width": float,
-      "isStored": Js.Nullable.t(bool),
-      "playableDuration": float,
-    },
-    "timestamp": float,
-    "location":
-      Js.Nullable.t({
-        .
-        "latitude": Js.Nullable.t(float),
-        "longitude": Js.Nullable.t(float),
-        "altitude": Js.Nullable.t(float),
-        "heading": Js.Nullable.t(float),
-        "speed": Js.Nullable.t(float),
-      }),
-  },
+type photoIdentifier = {note}
+and note = {
+  [@bs.as "type"]
+  _type: string,
+  [@bs.as "group_name"]
+  groupName: string,
```

Comment:
Honestly, I did not give it much thought. But we might want to change back since this a breaking change. I just changed it to pascalCasing to be constant with the rest of code.

Reply:
Not a problem to be a breaking change. It's breaking with or without the case change. It's more a general approach that we should think about: do we want to be closer to original naming or do we accept such adaptations in this bindings? cc @sgny @cknitt

Reply:
In the [community documentation](https://github.com/react-native-community/react-native-cameraroll#getphotos), these fields are in snake_case. It makes more sense to me now that I think about it to be consistent with the documentation. I change these back.